codespell_lib/data/dictionary.txt

```diff
@@ -25087,6 +25195,7 @@ positionns->positions
 positionof->position, position of,
 positiv->positive
 positivies->positives
+positivy->positivity, positivly, positive,
```

Comment:
Suggested word `positivly` is a misspelling. Please help to correct it.
```text
positivy->positivity, positively, positive,
```
* https://github.com/search?q=positivy&type=code  297